plotsquared/plot-flags.md

```diff
@@ -0,0 +1,286 @@
+
+Setting a flag is easy, just use `/plot flag set <flag> <value>`. For a boolean flag, the value would either be 'true' or 'false'.
+
+## Flag List
+
+To bring up a list of flags use `/plot flag list`. These values are clickable, and clicking a flag name will show you information about that particular flag. You can also use `/plot flag info <flag>` directly, if you already know the flag name.
+
+![Flag system](https://i.imgur.com/uTPjJEE.png)
+
+## Default Plot and Road Flags
+
+You can set default flags per world in the `plugins/PlotSquared/config/worlds.yml`. Default flags will be present in each claimed plot unless a different value is manually set for that plot.
+
+The following is an **example** of setting the `use` flag to allow interaction with doors, and trapdoors, levers, gates, pressure plates, buttons and trip wires:
+
+```yaml
+worlds:
+  <world>:
+    [...]
+    flags:
+      use: "oak_door,iron_trapdoor,lever,acacia_fence_gate,light_weighted_pressure_plate,oak_button,tripwire"
+      pvp: true
+```
+
+_`<world>` indicates the name of that plotworld._
+
+You can use [Vanilla block tags](/plotsquared/vanilla-tags.md) too.
+
+The same works for road flags.
+
+**Example:**
+
+```yaml
+worlds:
+  <world>:
+    [...]
+    road:
+      [...]
+      flags:
+        use: "oak_door,iron_trapdoor,lever,acacia_fence_gate,light_weighted_pressure_plate,oak_button,tripwire"
+        pvp: true
+```
+
+_`<world>` indicates the name of that plotworld._
+
+## Flag types
+
+### BlockTypeList
+
+A list of blocks, supporting both legacy numerical IDs and new string IDs e.g.
+
+* `grass_block` for grass blocks
+* `7` for bedrock
+* `7,3` for bedrock and dirt
+* `35:14,35:15` for red and black wool
+
+These flags also support [vanilla-tags](/plotsquared/vanilla-tags), like:
+
+* `#signs` for all sign types
+* `#buttons` for all button types
+
+### Boolean
+
+Accepts `true` or `false`.
+
+### Double
+
+Accepts any decimal number (all real numbers).
+
+### Enum
+
+Accepts a predefined collection of values.
+
+### Integer
+
+Accepts all integers.
+
+### Internal
+
+Flags used internally.
+
+### Miscellaneous
+
+Accept commands with multiple parameters, e.g `/gamemode creative`.
+
+### String
+
+Accept flags with multiple string parameters, e.g. `/plot f set greeting Welcome to my plot!`
+
+### TeleportDeny
+
+Accepts the follow plot membership `trusted`, `members`, `nonmembers`, `nontrusted` and `nonowners`.
+
+### Timed
+
+A binary tuple of numbers e.g. `53 6` or `7 93`.
+
+## Flag Permissions
+
+### Basic Permission Schema
+
+To give a player access to a flag, you need to assign the following permissions, and exchange the value if wanted:
+
+* `plots.set.flag`
+* `plots.flag`
+* `plots.flag.add`
+* `plots.flag.remove`
+
+{% hint style="info" %}
+Also, you can assign [`plots.permpack.basicflags`](/plotsquared/permission/permission-packs#_plots_permpack_basicflags) to cover the following permissions and lots of the basic flags.
```

Comment:
Clicking on the hyperlink should link within the document on gitbook. Currently, it links to GitHub.